eventdata/challenges/elasticlogs-querying.json

```diff
@@ -1,3 +1,5 @@
+{% set p_time_period = (time_period | default(1800)) %}
+
 {
   "name": "elasticlogs-querying",
   "description": "This challenge simulates a set of Kibana queries against historical data ({{p_query_index_pattern}} indices) without any indexing taking place for a period of 30 minutes. It assumes one of the challenges creating {{p_query_index_pattern}} indices has been run.",
```

Comment:
IMHO we should change the description to use the parameter, i.e. it should say `for a period of {{ time_period }} seconds` instead of `for a period of 30 minutes`.

Reply:
totally missed this, ty